kwargs.jl

```diff
@@ -5,14 +5,21 @@ import Pkg
 include(joinpath(@__DIR__, "autodetect-dependabot.jl"))
 
 function kwargs(; coverage::Bool,
-                  force_latest_compatible_version::Union{Bool, Symbol})
+                  force_latest_compatible_version::Union{Bool, Symbol},
+                  julia_args::Union{Cmd, AbstractVector{<:AbstractString}}=``)
     if !(force_latest_compatible_version isa Bool) && (force_latest_compatible_version != :auto)
         throw(ArgumentError("Invalid value for force_latest_compatible_version: $(force_latest_compatible_version)"))
     end
 
     kwargs_dict = Dict{Symbol, Any}()
     kwargs_dict[:coverage] = coverage
 
+    if VERSION >= v"1.6.0"
+        kwargs_dict[:julia_args] = julia_args
+    elseif julia_args != ``
+        @warn("The `julia_args` option requires at least Julia 1.6", VERSION, julia_args)
+    end
```

Comment:
```suggestion
    elseif julia_args != ``
        println("::warning::The `julia_args` option requires at least Julia 1.6. VERSION=$VERSION, julia_args=$julia_args")
    end
```

I think this will more noticable if it's a proper GHA-style warning. Doesn't seem worth it to install the GHA.jl package solely for the logger, so I'm using the [warning syntax](https://docs.github.com/en/actions/reference/workflow-commands-for-github-actions#setting-a-warning-message) manually.